### ISSUE TITLE
Error installing the library

### ISSUE THREAD

**Issue by eladbitton** [mduggan/cdifflib#4]
I am using a new docker container of ubuntu 18.04. 
I tried installing the library in two ways:
```
FROM ubuntu:18.04

WORKDIR /home/main
COPY . .

RUN apt-get update
RUN apt-get upgrade -y

RUN apt-get install python3.6 -y
RUN apt-get install python3-pip -y
RUN apt-get install python3-dev -y

RUN apt-get install build-essential -y
RUN git clone https://github.com/mduggan/cdifflib.git
RUN cd ./cdifflib && python3 setup.py install
```

Then i try i ssh to the machine and start python3:
```
Python 3.6.6 (default, Sep 12 2018, 18:26:19)
[GCC 8.0.1 20180414 (experimental) [trunk revision 259383]] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import cdifflib as s
>>> dir(s)
['__builtins__', '__cached__', '__doc__', '__file__', '__loader__', '__name__', '__package__', '__path__', '__spec__']
```

As you can see the classes of cdifflib are not here, so imports like CSequenceMatcher does not work.

The other way i tried is using pip:
`pip3 install cdifflib`

Which produced:
```
Collecting cdifflib
  Downloading https://files.pythonhosted.org/packages/51/4a/7f8b9102fdee98fb08ed978749002adba9167499463b3bb0b5f91c156106/cdifflib-1.1.0.tar.gz
Building wheels for collected packages: cdifflib
  Running setup.py bdist_wheel for cdifflib ... error
  Complete output from command /usr/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-s2vl2d1s/cdifflib/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" bdist_wheel -d /tmp/tmpt75e02rppip-wheel- --python-tag cp36:
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build/lib.linux-x86_64-3.6
  copying cdifflib.py -> build/lib.linux-x86_64-3.6
  running build_ext
  building '_cdifflib' extension
  creating build/temp.linux-x86_64-3.6
  x86_64-linux-gnu-gcc -pthread -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -I/usr/include/python3.6m -c _cdifflib3.c -o build/temp.linux-x86_64-3.6/_cdifflib3.o
  x86_64-linux-gnu-gcc: error: _cdifflib3.c: No such file or directory
  x86_64-linux-gnu-gcc: fatal error: no input files
  compilation terminated.
  error: command 'x86_64-linux-gnu-gcc' failed with exit status 1

  ----------------------------------------
  Failed building wheel for cdifflib
  Running setup.py clean for cdifflib
Failed to build cdifflib
Installing collected packages: cdifflib
  Running setup.py install for cdifflib ... error
    Complete output from command /usr/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-s2vl2d1s/cdifflib/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-u___gh9_-record/install-record.txt --single-version-externally-managed --compile:
    running install
    running build
    running build_py
    creating build
    creating build/lib.linux-x86_64-3.6
    copying cdifflib.py -> build/lib.linux-x86_64-3.6
    running build_ext
    building '_cdifflib' extension
    creating build/temp.linux-x86_64-3.6
    x86_64-linux-gnu-gcc -pthread -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -I/usr/include/python3.6m -c _cdifflib3.c -o build/temp.linux-x86_64-3.6/_cdifflib3.o
    x86_64-linux-gnu-gcc: error: _cdifflib3.c: No such file or directory
    x86_64-linux-gnu-gcc: fatal error: no input files
    compilation terminated.
    error: command 'x86_64-linux-gnu-gcc' failed with exit status 1

    ----------------------------------------
Command "/usr/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-s2vl2d1s/cdifflib/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-u___gh9_-record/install-record.txt --single-version-externally-managed --compile" failed with error code 1 in /tmp/pip-build-s2vl2d1s/cdifflib/
```

**Comment by mihirkhandekar** [comment]
I am also facing the same issue on Ubuntu 18.04 on Virtual Box with Python 3.6.7.

**Comment by mduggan** [comment]
Thanks for the reminder, I failed to look into this before but I worked out what the problem was just now - the `setup.py` is kinda wrong because it only includes the source file which is correct for whatever platform I build on - since I built the wheel on Py2 it didn't include the py3 source.  I have a simple fix that I think will work and will release it tomorrow after a bit of testing. For now the workaround would be to check out the source and just run `python setup.py install` manually, which I realise isn't the best.
